src/Components/FriendsPage/FriendsPage.js

```diff
@@ -1,41 +1,75 @@
-import React, { useState } from 'react'
-import './FriendsPage.css'
+import React, { useState, useEffect } from 'react';
+import './FriendsPage.css';
+import { getFriends } from '../../ApiCalls';
 
 function FriendsPage() {
+  const [search, setSearch] = useState('');
+  const [friends, setFriends] = useState([]);
+  const [filteredFriends, setFilteredFriends] = useState([]);
+  const [newFriendName, setNewFriendName] = useState('');
 
-  const friends = ['Trevor', 'Gus', 'Joe', 'Jill']
-  const [search, setSearch] = useState('Search Friends');
-  const [filteredFriends, setFilteredFriends] = useState(['Trevor', 'Gus', 'Joe', 'Jill']);
+  useEffect(() => {
+    getFriends(1)
+      .then(data => {
+        setFriends(data.data.friends);
+        setFilteredFriends(data.data.friends);
```

Comment:
This fetch shouldn't be setting state locally - it should be talking with the Redux root slice and assigning state there. We'll need to change this prior to merging - let me know if you want to chat through implementation - happy to help :)